WEBROOOOT/portfolio.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	
+	<link rel="stylesheet" href="style.css">
+
+	 <title>P0RTFOLIO</title>
+	</head>
+<nav> 
+
+		<a href="index.html">(It's All) ALL ABOUT ME</a> 
+	 	<a href="portfolio.html">PORTF0LIO</a> 
+
+
+	 </nav>
+
+</center>
+
+</head>
+
+<body>
+
+<h1> <img src="77eyes.gif"></h1>
+<br> 
+<br>
+<br>
+<h2><img src="LILSPARKLYTREE.gif"></h2>
+
+ <h3> ///SOME RECENT PR0JECTS/// </h3>
+
+ 	<style> 
+ 	p {color: black;}
+ <p> 
+Works 2010-2014-ish leaning largely towards the retinal 
+
+</p>
+
+
+	</style>
+
+		<img src="projects.png">
+
+
+<div class="line-separator"> </div>
```

Comment:
Nice way of adding a line in here (and nice use of a class)!  

An alternative way of adding a line separator is by using the ```<hr>``` tag.  (hr is short for "horizontal rule".)  You can style it however you'd like to in your CSS.